charts/longhorn/latest/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: longhorn
-version: 1.1.2
-appVersion: v1.1.2
-kubeVersion: ">=v1.16.0-r0"
+version: 1.2.0
+appVersion: v1.2.0
+kubeVersion: ">=v1.18.0"
```

Comment:
May need to change this to `"kubeVersion: >=1.18.0-r0"`

See https://github.com/longhorn/longhorn/issues/2963